newjersey/navigator.business.nj.gov
# feat: [#188489104] anytime action dropdown categories

## Description


We wanted to add categories to anytime action dropdown



### Ticket



This pull request resolves [#188489104](https://www.pivotaltracker.com/story/show/188489104).

### Approach

We took some time to refactor the how the CMS is structured to better represent these new categories which is a large part of the file changes. 



### Steps to Test

Form as a business, Go through the show funding card too,

See the anytime action drop down, 

see the categories

you can also go to the dev modifyBusiness URL and then add a license as expired to see that the anytime action liense reinstatements still work. 

you can also go to the anytime action tasks themselves to make sure they still work.




### Notes



## Code author checklist
